packages/Modal/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@paprika/modal",
+  "version": "0.1.0",
+  "description": "Modal component",
+  "author": "@paprika",
+  "main": "lib/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/Modal"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "directories": {
+    "lib": "lib",
+    "test": "tests"
+  },
+  "peerDependencies": {
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "styled-components": "^4.4.1"
```

Comment:
Does it matter that this component has a higher version dependency than the rest? Should we be keeping them all in sync?

Reply:
Say user install `@paprika/takeover` and `@paprika/modal`.

Takeover require `styled-components@^4.2.0`
Modal require `styled-components@^4.4.1`

Peer dependencies will be resolved on user side.

So yarn/npm will install only one version of SC (compatible with all packages). So `styled-components@4.2.1` will be installed.

So I think it is fine to have different versions of SC in packages.

Reply:
only if those versions are compatible between them?, I wouldn't like to have a package with sc 4 and another with 3 and eventually other with 5, which would make it harder for us to help our consumers in an episode of not having the right version.

Reply:
> only if those versions are compatible between them?

Yes.

>  I wouldn't like to have a package with sc 4 and another with 3 and eventually other with 5, which would make it harder for us to help our consumers in an episode of not having the right version.

I definitely wouldn't like to have multiple different major SC versions too. But I think it is ok to have different SC versions which are compatible.